changelogs/fragments/stat_no_md5.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+    - removed previously deprecated `get_md5` option from M(stat) module.
```

Does the `M(...)` syntax actually work for changelog fragments? Also, it should be
```suggestion
    - removed previously deprecated ``get_md5`` option from M(stat) module.
```
since this is treated as restructured text.